percona-server-8.0/Dockerfile

```diff
@@ -108,6 +108,17 @@ RUN set -eux; \
 ENV CALL_HOME_OPTIONAL_PARAMS=" -s ${OS_VER}"
 
 COPY ps-entry.sh /docker-entrypoint.sh
+COPY telemetry-agent-supervisor.sh /usr/bin/
+RUN set -ex; \
+    chown  mysql /usr/bin/telemetry-agent-supervisor.sh; \
+    chown  mysql /usr/bin/percona-telemetry-agent; \
+    chown  mysql /usr/local/percona/telemetry/history; \
+    chmod ug+rwx /usr/bin/telemetry-agent-supervisor.sh; \
+    chmod -R go+w /var/log/percona
```

Comment:
what about install command?

Reply:
Do you mean, command to install percona-telemetry-agent? Since TA package is a hard dependency for Percona-Server, it will be installed along with PS. Hence I've not added a separate command to install TA.